java-squid/src/main/java/org/sonar/java/se/ExplodedGraphWalker.java

```diff
@@ -412,21 +424,22 @@ private void executeAssignement(AssignmentExpressionTree tree) {
 
   private void executeArrayAccessExpression(ArrayAccessExpressionTree tree) {
     // unstack expression and dimension
-    Pair<ProgramState, List<SymbolicValue>> unstack = programState.unstackValue(2);
-    programState = unstack.a;
+    ProgramState.Pop unstack = programState.unstackValue(2);
+    programState = unstack.state;
     programState = programState.stackValue(constraintManager.createSymbolicValue(tree));
   }
 
   private void executeNewArray(NewArrayTree newArrayTree) {
-    programState = programState.unstackValue(newArrayTree.initializers().size()).a;
+    programState = programState.unstackValue(newArrayTree.initializers().size()).state;
     SymbolicValue svNewArray = constraintManager.createSymbolicValue(newArrayTree);
     programState = programState.stackValue(svNewArray);
     programState = svNewArray.setSingleConstraint(programState, NullConstraint.NOT_NULL);
   }
 
   private void executeNewClass(NewClassTree tree) {
+    // programState = constraintManager.processNewInstance(programState, tree);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)